pkg/util/globalerrors/errors.go

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package globalerrors
```

Comment:
I think this package name has to be singular (https://rakyll.org/style-packages/#:~:text=In%20go%2C%20package%20names%20are%20not%20plural.)

Reply:
Ok, will do. I named plural because of the golang `errors` package.